src/components/Flash/Flash.tsx

```diff
@@ -18,7 +19,7 @@ interface State {
 
 type MergedProps = Props & InjectedProps
 
-const REMOVE_DELAY = 8000
+const REMOVE_DELAY = 80000
```

Comment:
[ask] If you don't click `x`, it will remain for 80 seconds, is that correct?

Reply:
It was temporary change. I forgot 🙇